modules/apps/account/account-test/src/testFunctional/tests/Account.testcase

```diff
@@ -1649,6 +1649,10 @@ definition {
 
 			Click.clickAt(locator1 = "RolesPermissionsNavigation#PERMISSIONS_NAVITEM_SUBITEM_COLLAPSED");
 
+			if (IsElementNotPresent(key_subItem = "Account Management", locator1 = "RolesPermissionsNavigation#PERMISSIONS_NAVITEM_SUBITEM")) {
```

Comment:
Hello @melodywu,
If you look into the portal, you will find there is no need to do the collapse step for the Account Management, please try to find a new way to fix the issue. Thanks in advance,

Reply:
done